samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,225 @@
+		>
+			<Panel>
+				<Header title="First Panel" />
+				<Item onClick={goNext}>Second</Item>
+				<Item onClick={goThird}>Third</Item>
+			</Panel>
+			<Panel>
+				<Header title="Second Panel" />
+				<Item onClick={goPrevious}>First</Item>
+				<Item onClick={goNext}>Third</Item>
+			</Panel>
+			<Panel>
+				<Header title="Third Panel" />
+				<Item onClick={goFirst}>First</Item>
+				<Item onClick={goPrevious}>Second</Item>
+			</Panel>
+		</Panels>
+	);
+};
+
+
+storiesOf('Panels', module)
+	.add(
+		'Panels',
+		() => (<BasicPanels />),
+		{
+			props: {
+				noScroller: true,
+				noPanels: true
+			},
+			text: 'The basic Panels'
+		}
+	);
+
+const MenuItemBase = kind({
+	name: 'MenuItem',
+
+	propTypes: {
+		css: PropTypes.object,
+		icon: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+		path: PropTypes.string
+	},
+
+	defaultProps: {
+		disabled: false
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'menuItem'
+	},
+
+	handlers: {
+		onClick: handle(
+			forward('onClick')
+		)
+	},
+
+	computed: {
+		iconBefore: ({icon}) => {
+			if (!icon) return;
+
+			return (<Button size="large" backgroundOpacity="lightOpaque" icon={icon} />);
+		}
+	},
+
+	render: ({css, iconBefore, ...rest}) => {
+		return (
+			<Item
+				css={css}
+				{...rest}
+				slotAfter={
+					<Icon size="small">arrowlargeright</Icon>
+				}
+				slotBefore={iconBefore}
+			/>
+		);
+	}
+});
+
+const MenuItem = Linkable(MenuItemBase);
+
+const MainPanel = kind({
+	name: 'MainPanel',
+
+	render: (props) => (
+		<Panel spotlightId="main-panel-container" {...props}>
+			<Scroller>
+				<MenuItem path="./page1">Pages</MenuItem>
+				<MenuItem path="./page2">General</MenuItem>
+			</Scroller>
+		</Panel>
+	)
+});
+
+const Page1 = kind({
+	name: 'Page',
+
+	render: (props) => (
+		<Panel spotlightId="page1-container" {...props}>
+			<Scroller>
+				<MenuItem path="./endPage">Page1</MenuItem>
+				<MenuItem path="./endPage">Page2</MenuItem>
+			</Scroller>
+		</Panel>
+	)
+});
+
+const Page2 = kind({
+	name: 'Page',
+
+	render: (props) => (
+		<Panel spotlightId="page2-container" {...props}>
+			<Scroller>
+				<MenuItem path="./endPage">Page1</MenuItem>
+				<MenuItem path="./endPage">Page2</MenuItem>
+			</Scroller>
+		</Panel>
+	)
+});
+
+const EndPage = kind({
+	name: 'Page',
+
+	render: () => (
+		<div>
+			<h1>EndPage</h1>
+		</div>
+	)
+});
+
+const BreadcrumbPanelsBase  = kind({
+	name: 'Panels',
+	propTypes: {
+		onNavigate: PropTypes.func
+	},
+	render: ({onNavigate, ...rest}) => {
+		return (
+			<BreadcrumbPanels {...rest} onSelectBreadcrumb={onNavigate} />
+		);
+	}
+});
+
+const RoutablePanels = Routable({navigate: 'onNavigate'}, BreadcrumbPanelsBase);
+
+const App = class extends React.Component {
+	static displayName = 'App';
+
+	constructor (props) {
+		super(props);
+		this.state = {appPath: '/settings'};
+	}
+
+	onNavigate = ({path}) => {
+		this.setState({appPath: path});
+	};
+
+	render () {
+		return (
+			<div {...this.props}>
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/109738630-8141f500-7c0b-11eb-8ed0-ca86ad152b04.png)

 Unlike other samples, the X button is overlapped.

Reply:
I disabled the close button for BreadcrumbsPanel